util/merkle/src/lib.rs

```diff
@@ -109,3 +135,87 @@ pub trait TrieMut {
     /// value. Returns the old value associated with this key, if it existed.
     fn remove(&mut self, key: &[u8]) -> Result<Option<DBValue>>;
 }
+
+
+/// Trie types
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum TrieSpec {
+    /// Generic trie.
+    Generic,
```

Comment:
Why did you leave `Generic` instead of `Secure`?

Reply:
Secure will be removed, so I think i doesn't matter